pkg/cloudprovider/providers/openstack/openstack_loadbalancer.go

```diff
@@ -411,8 +425,8 @@ func (lbaas *LbaasV2) EnsureLoadBalancer(clusterName string, apiService *api.Ser
 
 		waitLoadbalancerActiveProvisioningStatus(lbaas.network, loadbalancer.ID)
 
-		for _, nodeName := range nodeNames {
-			addr, err := getAddressByName(lbaas.compute, types.NodeName(nodeName))
```

Comment:
If this returned hostname, and it translates to internal ip, we are good to go. Otherwise, do we need to mend firewall rules for lb -> node traffic, now that we're potentially using a different ip?